src/components/SocialButtons/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { IconButton, HStack } from '@chakra-ui/react'
+import { useRouter } from 'next/router'
+import { FaFacebook, FaInstagram, FaTwitter, FaYoutube } from 'react-icons/fa'
+import { IconType } from 'react-icons/lib'
+const SocialButton = ({
+  icon: Icon,
+  href,
+  label,
+}: {
+  icon: IconType
+  label: string
+  href: string
+}) => {
+  return (
+    <IconButton
+      as="a"
+      size="sm"
+      aria-label={label}
+      icon={<Icon />}
+      href={href}
+      variant="outline"
+      colorScheme="primary"
+    />
+  )
+}
+type SocialLinkType = {
+  label: string
+  icon: IconType
+  link: {
+    en: string
+    tr: string
+    nl: string
+  }
+}
+const SOCIAL_LINKS: SocialLinkType[] = [
```

Comment:
Just a comment: you may consider moving this kind of constant links to routes folder.